src/backbone.service.js

```diff
@@ -1,11 +1,39 @@
-import classify from 'backbone-metal-classify';
 import Radio from 'backbone.radio';
 import _ from 'underscore';
 import PromisePolyfill from 'es6-promise';
 
 const resolved = PromisePolyfill.Promise.resolve();
 
-Radio.Channel = classify(Radio.Channel);
+//copy of Backbone.js extend function
+Radio.Channel.extend = function(protoProps, staticProps) {
```

Comment:
Or just use `_.extend(Channel.prototype, {[methods]})`

The only use for classify here is calling _super that can be done with apply